src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -722,6 +722,19 @@ public void deleteUser(long id) {
         complete(submit(req("DELETE", tmpl("/users/{id}.json").set("id", id)), handleStatus()));
     }
 
+    public User suspendUser(User user, boolean action) {
```

Comment:
suspendUser and unsuspendUser? Tough one, method named suspend user to unsuspend doesn't seem logical.